modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/ServiceDetailsViewManager.java

```diff
@@ -199,7 +214,69 @@ public List<DeployedService> listDeployedServicesOfIsv(Category category, Csp cs
         String namespace = userServiceHelper.getCurrentUserManageNamespace();
         query.setNamespace(namespace);
         List<ServiceDeploymentEntity> deployServices = serviceDeploymentStorage.listServices(query);
-        return deployServices.stream().map(EntityTransUtils::convertToDeployedService).toList();
+        return setServiceConfigurationForDeployedService(deployServices);
+    }
+
+    private void setServiceConfigurationDetailsForDeployedService(DeployedService details,
+            ServiceDeploymentEntity serviceDeployment) {
+        ServiceTemplateEntity serviceTemplate = serviceDeployment.getServiceTemplateEntity();
+        if (Objects.nonNull(serviceTemplate)) {
+            ServiceConfigurationManage serviceConfigurationManage =
+                    serviceTemplate.getOcl().getServiceConfigurationManage();
+            if (Objects.nonNull(serviceConfigurationManage)) {
+                if (serviceTemplate.getServiceRegistrationState()
+                        .equals(ServiceRegistrationState.APPROVAL_PENDING)) {
+                    serviceTemplate.setReviewComment(ServiceRegistrationState.APPROVED.toValue());
+                    serviceTemplate.setServiceRegistrationState(ServiceRegistrationState.APPROVED);
+                    serviceTemplateStorage.storeAndFlush(serviceTemplate);
+                }
+                List<ServiceConfigurationParameter> configurationParameters =
+                        serviceConfigurationManage.getConfigurationParameters();
+                ServiceConfigurationDetails serviceConfigurationDetails =
+                        getServiceConfigurationDetails(serviceDeployment.getId(),
+                                configurationParameters, serviceTemplate.getLastModifiedTime());
+                details.setServiceConfigurationDetails(serviceConfigurationDetails);
+            }
+        }
+    }
+
+    private ServiceConfigurationDetails getServiceConfigurationDetails(UUID serviceId,
+            List<ServiceConfigurationParameter> parameters, OffsetDateTime updateTime) {
+        ServiceConfigurationDetails details = new ServiceConfigurationDetails();
+        details.setServiceId(serviceId);
+        Map<String, Object> configurationParameterMap = new HashMap<>();
```

Comment:
this does not match to what we discussed. 
We take what is available in the 'SERVICE_CONFIGURATION' and then merge it with values from the service entity.